src/motor_control.c

```diff
@@ -51,14 +58,7 @@ static inline void reset_tone(MotorControl *mc) {
     mc->tone_high = 0;
 }
 
-void motor_control_apply(MotorControl *mc, float abs_erpm, RunState state, float time) {
-    if (mc->parking_brake_mode == PARKING_BRAKE_ALWAYS ||
-        (mc->parking_brake_mode == PARKING_BRAKE_IDLE && state != STATE_RUNNING && abs_erpm < 50)) {
-        mc->parking_brake_active = true;
-    } else if (mc->parking_brake_mode == PARKING_BRAKE_NEVER || state == STATE_RUNNING) {
-        mc->parking_brake_active = false;
-    }
-
+void motor_control_apply(MotorControl *mc, MotorData *md, RunState state, float time) {
```

Comment:
Can you make it `const MotorData *md`? That ensures the `MotorData` struct can't be modified in the function, it's very helpful to enforce this and anyone reading the code can have immediate certainty the struct is not being messed with here.